packages/integration-karma/test/template/properties/index.spec.js

```diff
@@ -16,6 +16,7 @@ describe('live properties', () => {
             expect(elm.shadowRoot.querySelector('input').checked).toBe(true);
         });
 
+        // This test is relying in the fact we force re-rerender when api values change.
```

Comment:
@caridy this test specifically relies in that. im thinking that is used in the lightning-input.

Reply:
that's what I thought. lets chat about that with @sfdciuie tomorrow.

Reply:
The real-world scenario is for inputs that are in a badInput state, for example if you have a  type="date" or type="number" there may be text/content in the input, but the value would be empty `''`, setting the value to the same value of `''` should reset the input content (just as it does on a native input)

Reply:
@jodarove can you add the real-world scenario in the comment?

Reply:
@sfdciuie maybe in that case we can change it two two values... to trigger the reactivity. Saying, if input detects that the value is the same, it can set it to undefined and back to the original value in the same tick, if you know that setting the value will not no implications, it should be fine to set it twice since the rehydration will happen after in another tick.